myfirstcatapi/views/cat_view.py

```diff
@@ -79,3 +82,30 @@ async def list_cats(
     return dto.ListResponse[dto.CatSummary](
         results=cat_summary_list_response, metadata=cats.metadata
     )
+
+
+@router.get(
+    "/delete-cat/{cat_id}",
+    response_model=dto.UpdatedCount,
+    response_model_exclude_unset=True,
+)
```

Comment:
According to REST API "conventions", we should use the proper HTTP methods where applicable. 
```suggestion
@router.delete(
    "/cats/{cat_id}",
    response_model=dto.UpdatedCount,
    response_model_exclude_unset=True,
)
```
Some reading on this: https://restfulapi.net/http-methods/#delete

WDYT?

Reply:
done